app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
```

Comment:
I'd like to see a way to create logs that is a little bit less error prone. Maybe have a Log::create() function where you have to pass all those arguments. That way there is a single source of truth in setting the arguments.

Reply:
Or maybe a `$mensa->log(...)` :o

Reply:
what would be your minimum required arguments and optional arguments?

Reply:
I guess
required: severity, category, text
optional: user_id

Reply:
Although, `category` wouldn't be necessary as this is provided in mensa itself I guess